### PR TITLE
Add documentation on subtitles

### PR DESCRIPTION
**Pull request by Arnei**
*This PR aims at contributing to the implementation of #4407, but touches on just one of the aspects to realize #4407's vision. It should therefore likely not be merged alone, but alongside other PRs.*

Adds a new documentation file about subtitles.

### CODE REVIEW

File: docs/guides/admin/docs/modules/subtitles.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+be stored in the same flavor, while being optionally differentiated by [tags](#tags).
+
+### Tags
+
+Tags are used to hold additional information about individual subtitle tracks. All tags are optional and components
+should work without them being present, falling back to generic displays, or not showing information at all.
+
+The following is a list of tags that Opencast components may recognize:
+- `lang:<language>`
+	- `<language>` represents a language code of the form TODO: WHICH LANGUAGE CODE?
```
Comment (KatrinIhler) [inline docs/guides/admin/docs/modules/subtitles.md]: I assume these Todos should be done before we merge this?

Comment (Arnei) [reply docs/guides/admin/docs/modules/subtitles.md]: Thanks for the reminder.

Regarding language codes: I don't have a solid opinion on which to use. Do you? I should probably bring this is up in the Technical Meeting.

Regarding no major language: Might just have to ignore that for now and cross that bridge when we come to it?

Comment (KatrinIhler) [reply docs/guides/admin/docs/modules/subtitles.md]: > Regarding language codes: I don't have a solid opinion on which to use. Do you?

I think I prefer including the region in the language code, so `en-GB` instead of `en`. Most people commenting on the issue also seemed to think so.

> I should probably bring this is up in the Technical Meeting.

Good idea.

> Regarding no major language: Might just have to ignore that for now and cross that bridge when we come to it?

Agreed.

File: docs/guides/admin/docs/modules/subtitles.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
+Subtitles
+=========
+
+This document records guidelines and conventions on how subtitles should be handled in Opencast.
+
+<div class=warn>
+These guidelines are new to Opencast 14. Not all parts of Opencast may yet adhere to them!
```
Comment (mliradelc) [inline docs/guides/admin/docs/modules/subtitles.md]:
```suggestion
These guidelines are new to Opencast 15. Not all parts of Opencast may yet adhere to them! 
```

**Review by mliradelc — APPROVED** [review]
Docs are ok and very easy to undestand. Only a little update regarding opencast version

File: docs/guides/admin/docs/modules/subtitles.md (hunk 2 above)
Comment (mliradelc) [reply docs/guides/admin/docs/modules/subtitles.md]: Or maybe rewording to avoid update every time OC upgrades.

Comment (Arnei) [reply docs/guides/admin/docs/modules/subtitles.md]: Thanks for catching that again :see_no_evil:  
Maybe it's actually good having to look at this every major OC upgrade, so we're forced the re-evaluate if this warning is still appropriate?

Comment (mliradelc) [reply docs/guides/admin/docs/modules/subtitles.md]: I don't know, It's straightforward easy to forget to update the tiny details in the docs. I just found in the dev docs some repos to Bitbucket that are from almost 5 years deprecated.

Comment (Arnei) [reply docs/guides/admin/docs/modules/subtitles.md]: Well, in that case the version number at least serves as an indicator that this part of the docs is likely outdated, and as a user you should proceed with care :D

Comment (mliradelc) [reply docs/guides/admin/docs/modules/subtitles.md]: Fair Enough :)
